templates/zerver/register.html

```diff
@@ -116,8 +116,8 @@ <h1>{{ _('Create your account') }}</h1>
                             {{ _('Don&rsquo;t import settings') }}
                         </option>
                         {% for account in accounts %}
-                        <option value="{{ account.string_id }}" data-full-name="{{account.full_name}}" data-avatar="{{account.avatar}}"
-                          {% if ("source_realm" in form.data and account.string_id == form.data["source_realm"])
+                        <option value="{{ account.realm_id }}" data-full-name="{{account.full_name}}" data-avatar="{{account.avatar}}"
+                          {% if ("source_realm" in form.data and account.realm_id == form.data["source_realm"])
```

Comment:
We should also probably rename `source_realm` to `source_realm_id` as a separate follow-up commit so that it's clear we are dealing with `id`.